<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>formLogin</class>
    <widget class="QDialog" name="formLogin">
        <property name="enabled">
            <bool>true</bool>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>478</width>
                <height>681</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Inicio de sesión</string>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>10</y>
                    <width>291</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>18</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(33, 37, 41);</string>
            </property>
            <property name="text">
                <string>Inicio de sesión</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="logo">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>90</y>
                    <width>191</width>
                    <height>131</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: none;</string>
            </property>
            <property name="pixmap">
                <pixmap>../assets/logo_optica.png</pixmap>
            </property>
            <property name="scaledContents">
                <bool>true</bool>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtUsername">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>290</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string />
            </property>
        </widget>
        <widget class="QLineEdit" name="txtPassword">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>390</y>
                    <width>231</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    border: 1px solid rgb(173, 173, 173);
                    border-radius: 10px;
                    color: rgb(0, 0, 0);
                    padding: 5px;
                    background-color: rgb(255, 255, 255);
                </string>
            </property>
            <property name="echoMode">
                <enum>QLineEdit::Password</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string />
            </property>
        </widget>
        <widget class="QPushButton" name="btnLogin">
            <property name="geometry">
                <rect>
                    <x>160</x>
                    <y>480</y>
                    <width>171</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    background-color: rgb(111, 166, 80);
                    color: rgb(255, 255, 255);
                    border-radius: 8px;
                </string>
            </property>
            <property name="text">
                <string>Acceder</string>
            </property>
        </widget>
        <widget class="QPushButton" name="btnRegister">
            <property name="geometry">
                <rect>
                    <x>160</x>
                    <y>550</y>
                    <width>171</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(66, 134, 244);
                    color: rgb(255, 255, 255);
                    border-radius: 8px;

                </string>
            </property>
            <property name="text">
                <string>Registrarse</string>
            </property>
        </widget>
        <widget class="QLabel" name="lblError">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>630</y>
                    <width>361</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color: rgb(255, 85, 0);</string>
            </property>
            <property name="text">
                <string>Error</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblUsername">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>250</y>
                    <width>281</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Usuario</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="lblPassword">
            <property name="geometry">
                <rect>
                    <x>120</x>
                    <y>360</y>
                    <width>251</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
                    color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
                <string>Contraseña</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
    </widget>
    <resources>
        <include location="../logo.qrc" />
        <include location="logo" />
    </resources>
    <connections />
</ui>